Content.Client/_NF/ShuttleRecords/UI/ShuttleRecordsWindow.xaml

```diff
@@ -0,0 +1,70 @@
+﻿<controls:FancyWindow xmlns="https://spacestation14.io"
+                      xmlns:graphics="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
+                      xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+                      Title="{Loc 'shuttle-records-console-window-title'}"
+                      MinSize="700 300">
```

Comment:
If you increase the minimum size of the window, you avoid weird issues with the shuttle records window overlapping the top panel.
```suggestion
                      MinSize="700 510">
```